### PR TITLE
Update for Drupal 8.8.x compatibility.

### PR DESCRIPTION
**Pull request by grasmash**
acquia/d8-scaffold-files does not exist yet, but arguable it should soon.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,44 +5,44 @@
...
+            "docroot/profiles/contrib/{$name}": ["type:drupal-profile"],
+            "drush/Commands/contrib/{$name}": ["type:drupal-drush"],
         },
         "installer-types": [
             "bower-asset",
             "npm-asset"
-        ]
+        ],
+        "composer-scaffold": {
+            "locations": {
```
Comment (danepowell) [inline composer.json]: This can be removed, and instead require https://github.com/acquia/composer-scaffold

**Comment by danepowell** [comment]
If you want a template for setting this up, consider looking at https://github.com/acquia/blt-project

**Comment by balsama** [comment]
Closing in favor of #112

File: composer.json (hunk 1 above)
Comment (danepowell) [reply composer.json]: D'oh... I think I might be wrong here. Maybe you used to be able to delegate the location definitions, but no longer. They have to be in the root composer.json.

**Pull request closed without merging**